feature/platform/tests/power_admin_down_up_test/power_admin_down_up_test.go

```diff
@@ -31,7 +31,38 @@ func TestFabricPowerAdmin(t *testing.T) {
 				t.Skipf("Fabric Component %s is already INACTIVE, hence skipping", f)
 			}
 
+			before := intfOperStatus(t, dut)
+
 			powerDownUp(t, dut, f, oc.PlatformTypes_OPENCONFIG_HARDWARE_COMPONENT_FABRIC, 3*time.Minute)
+
+			start := time.Now()
+			after := intfOperStatus(t, dut)
+			for time.Since(start) < 3*time.Minute {
+				if len(after) == len(before) {
+					break
+				}
+				time.Sleep(30 * time.Second)
+				after = intfOperStatus(t, dut)
+			}
+			if len(after) != len(before) {
+				t.Errorf("Not all Interfaces came up after Power Down Up, got: %v, want: %v", after, before)
+			}
+
+			intfs := gnmi.WatchAll(t, dut, gnmi.OC().InterfaceAny().State(), 3*time.Minute, func(v *ygnmi.Value[*oc.Interface]) bool {
+				intf, ok := v.Val()
+				if !ok {
+					return false
+				}
+				op, ok := before[intf.GetName()]
+				if !ok {
+					return false
+				}
+				return intf.GetOperStatus() == op
+			})
+
+			if _, ok := intfs.Await(t); !ok {
+				t.Errorf("Interface Oper-Status didn't retun to original state after PowerAdmin Down Up for %s.", f)
```

Comment:
this logging statement is not helpful when the test fails, can u please change it to use

[featureprofiles/internal/helpers/helpers.go](https://github.com/openconfig/featureprofiles/blob/132572aa0c1c210a57642717de4bf86c57903d51/internal/helpers/helpers.go#L57)

Line 57 in [132572a](https://github.com/openconfig/featureprofiles/commit/132572aa0c1c210a57642717de4bf86c57903d51)
 func ValidateOperStatusUPIntfs(t *testing.T, dut *ondatra.DUTDevice, upIntfs []string, timeout time.Duration) { 
to log the exact interfaces which are not up after powerup.

Reply:
Using the helper now! Thanks for the pointer